### PR TITLE
Update Stackdriver event exporter version

### PR DESCRIPTION
**Pull request by crassirostris**
Fixes https://github.com/kubernetes/kubernetes/issues/50510

```release-note
fluentd-gcp addon: Fix a bug in the event-exporter, when repeated events were not sent to Stackdriver.
```

### CODE REVIEW

File: cluster/addons/fluentd-gcp/event-exporter.yaml [hunk 1]
```diff
@@ -29,10 +29,11 @@ subjects:
 apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
-  name: event-exporter-v0.1.4
```
Comment (piosz) [inline cluster/addons/fluentd-gcp/event-exporter.yaml]: will addona manager pick this change?

Comment (crassirostris) [inline cluster/addons/fluentd-gcp/event-exporter.yaml]: AFAIU yes, the old deployment will be deleted: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/addon-manager

> Addon will be deleted when its manifest file is deleted.

I think changing the name of the deployment is the same as removing the old addon and creating a new one

Comment (crassirostris) [inline cluster/addons/fluentd-gcp/event-exporter.yaml]: Just manually checked and it indeed works correctly

**Pull request merged**